client/webserver/site/src/js/app.ts

```diff
@@ -930,12 +921,25 @@ function setSeverityClass (el: HTMLElement, severity: number) {
   el.classList.add(severityClassMap[severity])
 }
 
+// Updates given order in market's orders list if it finds it.
+// Returns a bool which indicates if order was found.
+function updateOrder (mkt: Market, ord: Order) {
```

Comment:
I'm a noob at TypeScript but should this have the return type `boolean` at the end? Does it work correctly without it?

Reply:
typescript is able to tell by itself the return type by the function return. From the docs:

`We can add types to each of the parameters and then to the function itself to add a return type. TypeScript can figure the return type out by looking at the return statements, so we can also optionally leave this off in many cases.`

https://www.typescriptlang.org/docs/handbook/functions.html

But return type added. Thanks for the catch